instrumentation/opentelemetry/init.go

```diff
@@ -220,6 +220,15 @@ func InitWithSpanProcessorWrapper(cfg *config.AgentConfig, wrapper SpanProcessor
 	globalSampler = sampler
 	initialized = true
 
+	startSpanFn := startSpan(func() trace.TracerProvider {
+		return tp
+	})
+
+	// Startup span
+	_, span, ender := startSpanFn(context.Background(), "startup", &sdk.SpanOptions{})
+	span.SetAttribute("hypertrace.goagent.startup", true)
```

Comment:
Hmm, do you think this should be lang specific?  

`hypertrace.startup` or similar? 

Maybe not even valid concern but just to float it.
if a platform will have to look for these specific attributes something more generic could be better, so that we don't to update or maintain a separate list of `hypertrace.some-lang.startup` attrs to look for)

If we were just to filter on `hypertrace.startup` we could still tell languages apart by the `telemetry.sdk.language` attr.

Thoughts?

Reply:
Good call. I will rename to `hypertrace.agent.startup`